### ISSUE TITLE
Convert HTML file contents to ASCII doc

### ISSUE THREAD

**Issue by dannydeezy** [dannydeezy/bitcoin-book#1]
ASCII doc is a nice format that I think we should use (see https://github.com/bitcoinbook/bitcoinbook)

It would be nice to convert the HTML files to this type of doc.

If the HTML docs are too hard to parse through, you can use the threadreader app links or tweet threads themselves to construct

**Comment by Zero-1729** [comment]
I think this book is a great idea, the concepts are appropriately dissected to make it as easy as possible for people to understand. Great work @dannydeezy 👍

I am currently working on converting the html threads to ascii doc format, I'll open up a PR as soon as I'm done.

**Comment by dannydeezy** [comment]
Awesome! Looking forward to the PR. If I don't respond right away, feel free to ping me on twitter so I can approve it and get it merged

**Comment by dannydeezy** [comment]
@Zero-1729 I was able to get PDF files from ThreadReader - so feel free to use those if they help
https://github.com/dannydeezy/bitcoin-book/tree/master/thread-pdfs

**Comment by dannydeezy** [comment]
@Zero-1729 also don't worry about getting the GIFs working - I think these will all need to be replaced by static images anyway https://github.com/dannydeezy/bitcoin-book/issues/2

**Comment by Zero-1729** [comment]
@dannydeezy thanks for uploading the PDFs, they were quite useful. I also replaced the GIFs with static images (the specifics are in [my comment here](https://github.com/dannydeezy/bitcoin-book/issues/2#issuecomment-662120625)).

I am opening a PR now with all the changes.

**Comment by dannydeezy** [comment]
Awesome!!

**Comment by Zero-1729** [comment]
@dannydeezy I just opened the PR.
